src/plugins/uploader.js

```diff
@@ -0,0 +1,157 @@
+"use strict";
+
+const SocketIOFileUploadServer = require("socketio-file-upload/server");
+const Helper = require("../helper");
+const path = require("path");
+const fsextra = require("fs-extra");
+const fs = require("fs");
+const fileType = require("file-type");
+const readChunk = require("read-chunk");
+const crypto = require("crypto");
+const isUtf8 = require("is-utf8");
+const log = require("../log");
+
+const whitelist = [
+	"text/plain",
+
+	"image/png",
+	"image/jpeg",
+	"image/jpg",
+	"image/gif",
+	"image/bmp",
+	"image/webp",
+
+	"audio/midi",
+	"audio/x-midi",
+	"video/mp4",
+	"audio/mpeg",
+	"audio/mpeg3",
+	"audio/x-mpeg",
+	"audio/x-mpeg-3",
+	"audio/wave",
+	"audio/wav",
+	"audio/x-wav",
+	"audio/x-pn-wav",
+	"audio/webm",
+	"video/webm",
+	"audio/ogg",
+	"video/ogg ",
```

Comment:
Hey there's a space here

Reply:
I did this on purpose to differentiate. I can remove if youd prefer

Reply:
Differentiate what? Having a space in the string means this type won't be found.